internal/aws/ecsutil/testutil.go

```diff
@@ -0,0 +1,39 @@
+// Copyright  The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ecsutil
+
+import (
+	_ "embed"
+	"testing"
+)
+
+//go:embed testdata/container_metadata.json
```

Comment:
I believe this will cause this testdata and the testing module to get embedded into the production binary. Is that right? It's why a separate package would be safer I think, though I don't think it needs to be a separate go module.

Reply:
Seperate package makes sense to me, something of the name `ecsutiltest` would be clear that it is not intended for production use.

Reply:
I looked into this is a bit w/ hexdump and as far as I can tell the testdata ss only embedded in the production binary if the []byte vars are directly used in non-test code.  I still migrated the test helper to its own package though.